src/main/java/pl/javagame/Player.java

```diff
@@ -4,19 +4,16 @@
 
     String name;
 
-    public Player(String name) {
-try {
-    this.name = name;
-    if (this.name == null || this.name=="")
-        throw new Exception ("Podaleś null lub pusty text");
-}
-        catch (Exception exception) {
-            System.out.println(exception.getMessage());
+    public Player(String name)  {
+
+        if (name.isEmpty()){
```

Comment:
Dodatkowo jesli name jest nullem to poleci NullPointerException 'niekontrolowany'. Zdecydowanie lepiej zrobic `if(name==null || name.isEmpty()) { throw new IllegalArgument.... }`

Reply:
ok, dzięki